WordPress/openverse-frontend
# Fix filters spacing with a new header

## Fixes


Related to #1581 by @panchovm

## Description


This PR is a companion to #1931. It adds the correct padding values for the filters sidebar on `new_header` layouts. It fixes the lack of padding on the desktop screens:
<img width="333" alt="Screen Shot 2022-11-02 at 3 08 31 PM" src="https://user-images.githubusercontent.com/15233243/199486332-ee089cbb-2711-43e1-a783-aeec6c8b38b9.png">

Also, the unnecessary padding and margins are removed. To make sure that 'Clear filters' button doesn't affect the layout when it is shown or hidden, the button uses `absolute` positioning.

## Testing Instructions

Go to https://localhost:8443/preferences and make sure the New header flag is checked. Go to a search page and open the filters sidebar. It should have correct padding, and the layout should not change when 'Clear filters' button appears/disappears.

## Checklist


[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
